### ISSUE TITLE
Furious 0.4.7 Tun2socks problem.

### ISSUE THREAD

**Issue by OlroxIX** [LorenEteval/Furious#95]
Windows 11, Furious-0.4.7-windows10-amd64, vless connection Tun2socks stopped unexpectedly with exitcode 2. Furious 0.4.6 working without any issues.
Furious Log:
[Furious.QtFramework.CoreProcess] [INFO] Xray-core 1.8.23 started
[Furious.QtFramework.CoreProcess] [INFO] no core daemon for Xray-core in power save mode
[Furious.Utility.SystemRoutingTable] [INFO] delete rule 0.0.0.0->10.0.68.1 from routing table success. {'returncode': 0, 'stdout': '', 'stderr': 'The route deletion failed: Element not found.'}
[Furious.Utility.SystemRoutingTable] [INFO] get default gateway success. [('192.168.62.2', '192.168.62.128')]
[Furious.QtFramework.CoreProcess] [INFO] Tun2socks 2.5.2.1 started
[Furious.QtFramework.CoreProcess] [ERROR] Tun2socks stopped unexpectedly with exitcode 2
[Furious.Utility.SystemProxy] [INFO] turn off proxy success
[Furious.QtFramework.CoreProcess] [INFO] Xray-core terminated with exitcode -15

**Comment by furrycoding** [comment]
Had the same issue
First I tried different `wintun` versions - installing 0.11 and 0.13 made tun2socks return exitcode 1 instead

Then, after reinstalling wintun 0.14.1, I tried running `tun2socks` module manually like this:
(Python 3.11 running as admin, cd = where Furious.exe is)
```
>>> proxy = 'socks5://127.0.0.1:10809'
>>> import tun2socks.tun2socks
>>> tun2socks.tun2socks.startFromArgs('tun://test_tun', '', 'debug', proxy, '')
```
and got the following [log file](https://github.com/user-attachments/files/17300693/tun2socks.log)

It seems that the relevant part is
```
runtime: newstack at runtime.printlock+0x55 sp=0xc0000475f8 stack=[0xc000044000, 0xc000048000]

...

syscall.SyscallN(...?, {...?, 0x3?, ...?})
        runtime/syscall_windows.go:544 +0x107 fp=0xc0000477e8 sp=0xc000047770 pc=..
syscall.Syscall(...?, 0x68?, ...?, ...?, ...?)
        runtime/syscall_windows.go:482 +0x35 fp=0xc000047830 sp=0xc0000477e8 pc=...
golang.zx2c4.com/wintun.CreateAdapter({...?, 0x2c?}, {..., 0x9}, 0x0)
        golang.zx2c4.com/wintun@v0.0.0-20230126152724-0fa3db229ce2/wintun.go:95 +0xf9 fp=0xc0000478b8 sp=0xc000047830 pc=...
golang.zx2c4.com/wireguard/tun.CreateTUNWithRequestedGUID({..., 0x8}, ...?, 0x0)
        golang.zx2c4.com/wireguard@v0.0.0-20231211153847-12269c276173/tun/tun_windows.go:69 +0x45 fp=0xc000047928 sp=0xc0000478b8 pc=...
golang.zx2c4.com/wireguard/tun.CreateTUN(...)

...

fatal error: runtime: stack split at bad time
```

**Comment by LorenEteval** [comment]
@OlroxIX @furrycoding Please try [Furious 0.4.8](https://github.com/LorenEteval/Furious/releases/tag/0.4.8) with updated `tun2socks` library.

**Comment by OlroxIX** [comment]
Unfortunately with 0.48 result is the same as was with 0.47.

**Comment by somebodywashere** [comment]
Yes, I can confirm that 0.4.7 and 0.4.8 clean installations still not working with TUN mode.

0.4.6 works well

**Comment by spanchy** [comment]
0.4.6 - windows 11 TUN-mode not work.
0.4.7 - windows 11 TUN-mode not work
0.4.8 - windows 11 TUN-mode not work

**Comment by LorenEteval** [comment]
The `tun2socks` binding code remains the same in previous versions, only those dependencies in go library were updated. This needs time for debugging and hopefully I can reproduce the error logs provided by @furrycoding .

**Comment by LorenEteval** [comment]
Same as https://github.com/LorenEteval/Furious/issues/69. Closing
